tests/test_resource.py

```diff
@@ -346,6 +346,25 @@ def test_get_pagination(self):
             },
         }
 
+    def test_get_pagination_multifield_query_params(self):
+        expected_prefix = "/test/path?test=param&another=example&another=example2"
+        req.query_params = MultiDict(
+            {
+                "test": ["param"],
+                "another": ["example", "example2"],
+            }
+        )
+        self.resource.request = req
+        assert self.resource.get_pagination(total_items=100, offset=20, limit=10) == {
```

Comment:
Please, place an empty line before this one just to visually separate steps of this test 😉